openedx_webhooks/lib/github/test/utils/test_get_most_recent_hook.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import namedtuple
+
+import arrow
+import pytest
+
+from openedx_webhooks.lib.github.utils import _get_most_recent_hook
+
+Hook = namedtuple('Hook', ['active', 'updated_at'])
```

Comment:
It's confusing to have two different stub Hook namedtuples in two different files.  Could we have a single stub class to use everywhere?